muspinsim/muspinsim.xml

```diff
@@ -0,0 +1,62 @@
+<tool id="muspinsim" name="Simulate muon experiment spin dynamics (muspinsim)" version="0.1.0" python_template_version="3.5">
+    <requirements>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        pip install git+https://github.com/muon-spectroscopy-computational-project/muspinsim.git &&
+        out_base=\$(grep -A1 '^name' $mu_sim | grep -v 'name' | sed -e 's/^[ \t]*//') &&
+        out_base=\${out_base:-"muspinsim"} &&
+        log_file=\$(echo $mu_sim | cut -d '.' -f1).log &&
+        ln -s $mu_exp_in $mu_exp_in.name &&
+        if grep '^fitting_data' $mu_sim; then sed -iE "/^fitting_data/{n;s/([^)]*)/(\"$mu_exp_in.name\")/g}" $mu_sim; fi &&
+        muspinsim $mu_sim &&
+        mv \$out_base.dat mu_out.dat &&
+        mv \$log_file log_out.log
+    ]]></command>
+    <inputs>
+        <param type="data" name="mu_sim" label="input file containing simulation parameters"/>
+        <param type="data" name="mu_exp_in" optional="true" label="Experiment data to fit (.dat) (Optional)" />
+    </inputs>
+    <outputs>
+        <data label=".dat output file for $mu_sim.name" name="mu_out" format="txt" from_work_dir="mu_out.dat" />
+        <data label=".log output file for $mu_sim.name" name="log_out" format="txt" from_work_dir="log_out.log" />
+    </outputs>
+    <tests>
+        <test>
+            <param name="mu_sim" value="hfine.in" ftype="txt" />
+            <output name="mu_out" file="hfine.dat" ftype="txt" compare="sim_size" lines_diff="20" />
+            <output name="log_out" file="hfine.log" ftype="txt" compare="sim_size" delta="100" />
+        </test>
+        <test>
+            <param name="mu_sim" value="hfine_powder.in" ftype="txt" />
+            <output name="mu_out" file="hfine_powder.dat" ftype="txt" compare="sim_size" delta="20" />
+            <output name="log_out" file="hfine_powder.log" ftype="txt" compare="sim_size" delta="100" />
+        </test>
+        <test>
+            <param name="mu_sim" value="fitting.in" ftype="txt" />
+            <param name="mu_exp_in" value="fitting_input.dat" ftype="txt" />
+            <output name="mu_out" file="fitting_data.dat" ftype="txt" compare="sim_size" delta="20" />
+            <output name="log_out" file="fitting.log" ftype="txt" compare="sim_size" delta="100" />
+        </test>
+    </tests>
+    <help><![CDATA[
+        MuSpinSim is a program designed to carry out spin dynamics calculations
+
+        Tool requires an input (.in) file which describes system and experiment details
+        This is a simple text file containing keywords and values in indented blocks.
```

Comment:
I favour avoiding the word "simple" in help text/documentation in general, as it can be demotivating to new users who are struggling.

```suggestion
        This is a text file containing keywords and values in indented blocks.
```

Reply:
changes committed [3587b12](https://github.com/muon-spectroscopy-computational-project/muon-galaxy-tools/pull/55/commits/3587b129f90aa16b3f391cffbfd48d6f8288b495)